submission/persistence-mongo/src/main/kotlin/ac/uk/ebi/biostd/persistence/doc/db/repositories/Repositories.kt

```diff
@@ -35,7 +35,7 @@ interface SubmissionMongoRepository : MongoRepository<DocSubmission, ObjectId> {
 
     fun getByAccNoInAndVersionGreaterThan(accNo: List<String>, version: Int): List<DocSubmission>
 
-    fun findFirstByAccNoOrderByVersionDesc(accNo: String): DocSubmission?
+    fun findFirstByAccNoAndVersionLessThanOrderByVersion(accNo: String, version: Int = 0): DocSubmission?
```

Comment:
why sorting by AccNo?

Reply:
It's not sorting by accNo but by version